src/stories/Library/event-list/event-list.scss

```
@@ -7,3 +7,7 @@
   list-style: none;
   margin-bottom: $s-md;
 }
+
+.event-list__item--stacked {
+  margin-top: -$s-md;
+}
```

```suggestion
.event-list__item + .event-list__item--stacked,
.event-list__item + .event-list__item {
  margin-bottm: $s-md;
}
```


Instead of using -margin, i would suggest to remove the margin-bottom from .event-list-item, and then use what i wrote above. I think it accomplishes the same. Personal preference of only "adding" margin instead of adding and then withdrawing it again.